t5x/train.py

```diff
@@ -705,6 +702,9 @@ def _as_gda(spec):
 
   FLAGS = flags.FLAGS
 
+  # OOM fix. Prevents TF from seeing GPUs to stop conflict with JAX.
+  tf.config.experimental.set_visible_devices([], 'GPU')
```

Comment:
We have had an issue with OOM errors in eval/infer.
Does this line not need to be included there too?